### PR TITLE
Add dz60 iso layout with vim-style arrows

### PR DESCRIPTION
**Pull request by andys8**
I added a new layout for the dz60 board targeting iso layout. There are vim style arrow keys on the function layer and in the bottom right corner of the main layer.

I've tested it and am using it on my current keyboard.

### CODE REVIEW

File: keyboards/dz60/keymaps/iso_vim_arrow/keymap.c [hunk 1]
```diff
@@ -0,0 +1,68 @@
+#include "dz60.h"
```
Comment (drashna) [inline keyboards/dz60/keymaps/iso_vim_arrow/keymap.c]: If you could replace this with `#include QMK_KEYBOARD_H`?

File: keyboards/dz60/keymaps/iso_vim_arrow/keymap.c [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+* | Tab    |  Q  |  W  |  E  |  R  |  T  |  Z  |  U  |  I  |  O  |  P  |  P  |  Ü  | *  |   |
+* |-------------------------------------------------------------------------------------|   +
+* | Layer_1    |  A  |  S  |  D  |  F  |  G  |  H  |  J  |  K  |  L  |  Ö  |  Ä |  # |Enter |
+* |-----------------------------------------------------------------------------------------+
+* | Shift|  < |  Y  |  X  |  C  |  V  |  B  |  N  |  M  |  ,  |  .  |  .  |  -  |  RShift   |
+* |-----------------------------------------------------------------------------------------+
+* | LCtrl |  LGUI  |  LAlt  |          Space           |  RAlt | Left |  Down | Up  | Right |
+* `-----------------------------------------------------------------------------------------'
+*/
+	KEYMAP(
```
Comment (drashna) [inline keyboards/dz60/keymaps/iso_vim_arrow/keymap.c]: If you could replace this with `LAYOUT`?

File: keyboards/dz60/keymaps/iso_vim_arrow/keymap.c [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+		KC_ESC, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_NO, KC_BSPC,
+		KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, KC_BSLS,
+		MO(1), KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_ENT,
+		KC_LSFT, KC_NUBS, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_RSFT, KC_NO,
+    KC_LCTL, KC_LGUI, KC_LALT, KC_SPC, KC_SPC, KC_SPC, KC_RALT, KC_LEFT, KC_DOWN, KC_UP, KC_RIGHT),
+
+
+	KEYMAP(
```
Comment (drashna) [inline keyboards/dz60/keymaps/iso_vim_arrow/keymap.c]: If you could replace this with `LAYOUT`?

**Comment by andys8** [comment]
@drashna Thanks for the feedback. I made the requested changes. The layout can be compiled. I didn't flash the changes to the keyboard for now.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**